tests/anoph/test_frq.py

```diff
@@ -0,0 +1,98 @@
+import pytest
+import plotly.graph_objects as go  # type: ignore
+
+import random
+
+
+@pytest.mark.skip
+def test_plot_frequencies_heatmap(api, frq_df):
```

Comment:
Suggest to rename "check_..." instead of "test_...". That way the function won't be seen as a test by pytest, and so won't need to be marked for skipping. Also it will follow the convention used elsewhere in malariagen_data, where utility functions for use in testing are named "check_...".

Same comment applies to all functions in this module.

Reply:
Noting here that the code in this PR Is now:
```python
def check_plot_frequencies_heatmap(api, frq_df):
    fig = api.plot_frequencies_heatmap(frq_df, show=False, max_len=None)
    assert isinstance(fig, go.Figure)

    # Test max_len behaviour.
    with pytest.raises(ValueError):
        api.plot_frequencies_heatmap(frq_df, show=False, max_len=len(frq_df) - 1)

    # Test index parameter - if None, should use dataframe index.
    fig = api.plot_frequencies_heatmap(frq_df, show=False, index=None, max_len=None)

    if "contig" in list(frq_df.columns):
        # Not unique.
        with pytest.raises(ValueError):
            api.plot_frequencies_heatmap(
                frq_df, show=False, index="contig", max_len=None
            )
```